Makefile

```diff
@@ -68,6 +69,12 @@ modules: FORCE
 runtime: FORCE
 	cd runtime && $(MAKE)
 
+llvm-runtime-maybe: FORCE
```

Comment:
Then I should remove the hyphen before the if-statement and rename it to llvm-runtime-if-needed...